packages/shared/lib/services/connections/credentials/refresh.ts

```diff
@@ -81,6 +81,7 @@ export async function refreshOrTestCredentials(props: RefreshProps): Promise<Res
 
         // TODO: remove this when cron is using other columns
         await connectionService.updateLastFetched(props.connection.id);
+        props.connection = { ...props.connection, last_fetched_at: new Date() };
```

Comment:
Either that or we return the new connection and decrypt, wdyt?

Reply:
I like the idea of the db being the source of truth but the decrypting step seems unnecessary. How close of implementing the TODO are we? if we are gonna remove the code anyway I will keep it as it is

Reply:
not sure tbh, I still feel it could get removed but we need to store the refresh token expiration date to finish the migration